abp/templates/admin/module/angular/apps/dev-app/src/environments/environment.ts

```diff
@@ -22,7 +22,15 @@ export const environment = {
       url: 'https://localhost:44301',
       rootNamespace: 'MyCompanyName.MyProjectName',
     },
-    MyProjectName: {
+    MyProjectNameCommon: {
+      url: 'https://localhost:44300',
+      rootNamespace: 'MyCompanyName.MyProjectName',
+    },
+    MyProjectNameAdmin: {
+      url: 'https://localhost:44300',
+      rootNamespace: 'MyCompanyName.MyProjectName',
+    },
```

Comment:
Change root namespace as MyCompanyName.MyProjectName.Admin [for more](https://github.com/allegorysoft/library/blob/6a7c9dd505aec82982eaae686e1f2115987ef85b/abp/templates/admin/module/aspnet-core/src/admin/MyCompanyName.MyProjectName.Admin.Application/MyCompanyName.MyProjectName.Admin.Application.csproj#L7)